DocNet.Core/Markup/Views/assemblyViewTemplate.html

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<html>
+
+<head>
+<meta charset="utf-8">
+	<link href="../css/bootstrap.min.css" rel="stylesheet">
```

Comment:
We should pass the paths to the css files as arguments to the views. Makes changing names/paths easier.